collector/src/p2p/ingress.rs

```diff
@@ -46,7 +46,7 @@ impl<P: PublicKey, R: Committable + Digestible + Codec> Originator for Mailbox<P
                 responder: tx,
             })
             .await;
-        rx.await.unwrap()
+        rx.await.unwrap_or_default()
```

Comment:
Even if we want to add `Error` handling (like in #1549), we may still want handling like this...